docs/cicd-pipeline-migration.md

```
@@ -0,0 +1,176 @@
+```
+deploy.yml
+  → reusable-build-docker.yml (이미지 빌드 + ECR 푸시)
+  → reusable-deploy-ecs.yml (Task Definition 업데이트 + ECS 서비스 배포)
+  → 끝 (Terraform은 관여하지 않음)
+```
+
+---
+
+## 영향 범위
+
+### 영향 있음
```

markdownlint-cli2 (0.22.0)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)